test/examples/unified_field_list_examples/existing_fields.ts

```diff
@@ -73,7 +74,15 @@ export default ({ getService, getPageObjects }: FtrProviderContext) => {
       await PageObjects.timePicker.setAbsoluteRange(TEST_START_TIME, TEST_END_TIME);
       await PageObjects.header.waitUntilLoadingHasFinished();
       await PageObjects.unifiedFieldList.waitUntilSidebarHasLoaded();
-      await PageObjects.unifiedFieldList.toggleSidebarSection('meta');
+      await retry.waitFor('meta fields section to open', async () => {
+        await PageObjects.unifiedFieldList.toggleSidebarSection('meta');
+        return await find.existsByCssSelector(
+          `${PageObjects.unifiedFieldList.getSidebarSectionSelector(
+            'meta',
+            true
+          )}.euiAccordion-isOpen`
+        );
```

Comment:
Can we move it inside `toggleSidebarSection`?

Reply:
I considered this, but currently `toggleSidebarSection` can be used for both opening and closing sections, so I didn't want to change it to work for opening only. But I agree it makes sense to move the logic to the page object to reuse, so I added a new `openSidebarSection` method instead: https://github.com/elastic/kibana/pull/176972/commits/87e08df871c8e3b08bf76ccde0330de7a51da63a. Wdyt?